### PR TITLE
move current user to top of the list

### LINKED ISSUE #274: Need to have logged in user's name at the top of the list
- [ ] Need to have logged in user's name at the top of the list + some indication that it's them

### PR DESCRIPTION
**Pull request by Nazeeh21**
Closes #274

### CODE REVIEW

**Comment by farque65** [comment]
Not seeing the current user on top of the list

**Review by farque65 — COMMENTED** [review]
Not seeing the current user on top of the list

File: packages/react-app/src/views/GuestRoom.jsx [hunk 1]
```diff
@@ -41,6 +42,20 @@ export default function GuestRoom({
 
   const subs = useRef([]);
 
+  useEffect(() => {
+    // moving current user to the top of the list
+    if (addresses && addresses.length > 0) {
+      const newAddresses = [...addresses];
+      newAddresses.forEach((add, index) => {
+        if (add === address) {
```
Comment (varunv5000) [inline packages/react-app/src/views/GuestRoom.jsx:50]: Comparing it with lowercase might make it work:
(add.toLowerCase() === address.toLowerCase())

Comment (Nazeeh21) [reply packages/react-app/src/views/GuestRoom.jsx]: Thanks for suggesting. Added that

**Comment by Nazeeh21** [comment]
> Not seeing the current user on top of the list

@farque65 I think the problem would be the casing and I've updated that, so can you please check that again.

**Pull request merged**